db/migrate/20210111145461_change_accounts_username_to_be_nullable.openstax_accounts.rb

```diff
@@ -0,0 +1,6 @@
+# This migration comes from openstax_accounts (originally 8)
+class ChangeAccountsUsernameToBeNullable < ActiveRecord::Migration[4.2]
```

Comment:
some of these migrations don't seem like they belong to this project 🤔

Reply:
accounts-rails (openstax_accounts) and maybe openstax_salesforce gems have migrations. I believe this is because they are engines and not just gems.

Reply:
You could probably copy the final result of whatever is in the schema.rb at the end and put that in a migration to replace all of these accounts-rails migrations if you wanted.